ls1intum/Artemis
# [Bugfix] Add missing translations in quiz instructions

### Checklist
#### General


#### Server
#### Client

*not applicable
### Motivation and Context


Concerns #2424 

### Description

Currently, the quiz instructions occurring when adding a new quiz question where only available in English. This PR introduces the translation to German.

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Log in to Artemis
2. Navigate to Course Administration
3. Create a new Quiz exercise
4. Set your language settings to English
5. Add a MC/DnD/SA question to the exercise and review the creation instructions (should be the same as usual)
5.1. For MC and DnD questions, add hints and explanations via the corresponding buttons and make sure that the instructions appear in English
6. Delete the added questions
7. Change your language settings to German
8. Redo point 5 and 5.1. Make sure the instructions are now in German
9. Review the instructions in German. The translation so far is rather word for word, I'm open for feedback.

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage






### Screenshots


**German translation (in this order: MC-DnD-SA):**
![image](https://user-images.githubusercontent.com/80622272/140396036-83bd7738-7322-4deb-92aa-2b805feef050.png)
![image](https://user-images.githubusercontent.com/80622272/140396132-c94b5aca-f99f-434c-9cf3-b9f8304c541f.png)
![image](https://user-images.githubusercontent.com/80622272/140396211-d01a5c15-9597-40b1-b490-5823aad1cc8c.png)
